LLD/src/designpatterns/singleton/Singleton.java

```diff
@@ -0,0 +1,15 @@
+package designpatterns.singleton;
+
+public class Singleton {
+    private static Singleton instance = null;
+
+    public Singleton() {
```

Comment:
This must be private in Singleton design pattern
```suggestion
    private Singleton() {
```